eng/Version.Details.xml

```diff
@@ -8,9 +8,9 @@
       <Uri>https://github.com/dotnet/linker</Uri>
       <Sha>d0662ed8db919642177ddfd06a1c33895a69015f</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.1-mauipre.1.21616.2">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="6.0.2-mauipre.1.21622.4">
```

Comment:
@steveisok are we good to move to 6.0.2 in `main`? Does this match up with the 6.0.200 SDK?

Reply:
Equivalent iOS PR: https://github.com/xamarin/xamarin-macios/pull/13633

Reply:
Ok, I'm confused when I look at the diff: https://github.com/dotnet/runtime/compare/2ab2de4...37e3ee4

But I don't see those commits on the "MAUI" release branch: https://github.com/dotnet/runtime/commits/release/6.0-maui

Is this switching us to a different branch of dotnet/runtime?

Reply:
6.0.2-mauipre.1.21622.4 is this build: https://dev.azure.com/dnceng/internal/_build/results?buildId=1524265&view=results

Reply:
Which is this commit from the release/6.0-maui branch: https://github.com/dotnet/runtime/commit/37e3ee473ad4886ef758636c13dabf1ed4d3e73b